### PR TITLE
598 feed plugin image support

### PR DESCRIPTION
**Pull request by negrel**
## Description

Add image support to feed plugin.

## Related Issues

Addresses #598.

### Check List

- [ ] Have you read the
      [CODE OF CONDUCT](https://github.com/lumeland/lume/blob/master/CODE_OF_CONDUCT.md)
- [x] Have you read the document
      [CONTRIBUTING](https://github.com/lumeland/lume/blob/master/CONTRIBUTING.md)
  - [x] One pull request per feature. If you want to do more than one thing,
        send multiple pull request.
  - [x] Write tests.
  - [x] Run deno `fmt` to fix the code format before commit.
  - [x] Document any change in the `CHANGELOG.md`.

### CODE REVIEW

File: plugins/feed.ts [hunk 1]
```diff
@@ -154,6 +158,10 @@ export default function (userOptions?: Options) {
           const content = getDataValue(data, items.content)?.toString();
           const pageUrl = site.url(data.url, true);
           const fixedContent = fixUrls(new URL(pageUrl), content || "");
+          const imagePath = getDataValue(data, items.image);
+          const image = imagePath !== undefined
+            ? new URL(imagePath, site.url("", true)).toString()
```
Comment (oscarotero) [inline plugins/feed.ts:163]: Don't need to create a URL. This is simpler:
```suggestion
            ? site.url(imagePath, true))
```

**Review by oscarotero — CHANGES_REQUESTED** [review]

File: plugins/feed.ts (hunk 1 above)
Comment (negrel) [reply plugins/feed.ts]: done!

**Comment by oscarotero** [comment]
Thanks. The tests are failing due the new image file in the assets folder. Can you run `deno task test tests/feed.test.ts -- --update` to update it?

**Comment by negrel** [comment]
> Thanks. The tests are failing due the new image file in the assets folder. Can you run `deno task test tests/feed.test.ts -- --update` to update it?

It seems that my change to test site also impacted snapshot of other tests. I've updated snapshot of those test in a separate commit. I can't run test suite locally, favicon test seems to never end so some failing test may remain. Let me know if that's the case.

**Comment by oscarotero** [comment]
The `favicon` issue is a bug from Deno in its support for NPM packages (https://github.com/denoland/deno/issues/21686)

I'm using Deno 1.38.5 to run the tests because it's the most recent version without this bug. You can run `deno upgrade --version 1.38.5` to update the tests. Then back to the latest Deno version with `deno upgrade`.

**Comment by negrel** [comment]
Should be ok now.

**Comment by oscarotero** [comment]
Thank you!

**Pull request merged**